powerauth-webflow/pom.xml

```diff
@@ -32,6 +32,21 @@
     <description>Federated Authentication and Authorization Server for Web</description>
     <packaging>war</packaging>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-starter-tomcat</artifactId>
+                <scope>provided</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.tomcat.embed</groupId>
+                <artifactId>tomcat-embed-el</artifactId>
```

Comment:
I believe that when we declare `tomcat-embed-el` as _provided_, we could remove its exclusion elsewhere, couldn't we?